app/models/application_record.rb

```diff
@@ -0,0 +1,3 @@
+class ApplicationRecord < ActiveRecord::Base
```

Comment:
Can we check to make sure this is really a Rails 5 convention before commit to this rewiring? Right now this middle man class looks useless to the untrained eye.